components/atoms/ActionButton.js

```diff
@@ -0,0 +1,121 @@
+import PropTypes from "prop-types";
+
+/**
+ * Button component
+ */
+export function ActionButton(props) {
+  return (
+    <button
+      className={`flex justify-center content-center h-auto p-1 ${
+        props.fullyRounded
+          ? "rounded-full py-2 px-4"
+          : props.rounded
+          ? "rounded-md py-2 px-4"
+          : props.notRound
+          ? "rounded-sm"
+          : "rounded-md"
```

Comment:
We only have 1 style of button in the mockups, one with a small border radius. Can we take out the "rounded" option until it shows up?